src/containers/BlogPage/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Blogs from "../../components/Blogs";
+import SearchBar from "../../components/SearchBar";
+import "./index.scss";
+import { useTranslation } from "react-i18next";
+
+export default function BlogPage() {
+  const { t } = useTranslation();
+
+  return (
+    <div className="blogsMainContainer">
+      <h2 className="ourBlogTitle">Our Blog</h2>
+      <p className="ourBlogSubtitle">
+        {t(`Information about violence, women and health. Read different topics
```

Comment:
this is not right. you can check how we did translation before. you need to add this text to translation.json and then select it here.

Reply:
I wanted to do it in this way, but I wasn't sure if this will cause a conflict in the JSON file since Hiba was adding new data. So I kept it in this way till she finishes adding the data so that I can add to it later.

Reply:
just write to en.json first.